cider/tests/CiderBitUtilsTest.cpp

```diff
@@ -102,18 +102,23 @@ TEST_F(CiderBitUtilsTest, CountTest) {
   auto bit_vec = bit_vec_1.as<uint8_t>();
 
   EXPECT_EQ(countSetBits(bit_vec, 1025), 0ul);
+  EXPECT_EQ(countUnsetBits(bit_vec, 1025), 1025ul);
 
   setBitAt(bit_vec, 0);
   EXPECT_EQ(countSetBits(bit_vec, 1025), 1ul);
+  EXPECT_EQ(countUnsetBits(bit_vec, 1025), 1024ul);
 
   setBitAt(bit_vec, 1024);
   EXPECT_EQ(countSetBits(bit_vec, 1025), 2ul);
+  EXPECT_EQ(countUnsetBits(bit_vec, 1025), 1023ul);
```

Comment:
Based on definition of `countUnsetBits`, seems it's not necessary to this check, I mean, 1023 is always equal to 1025 plus 2.

Reply:
Yes. Should I add some different test cases or should I just remove these?

Reply:
I am not sure what you want to cover by these TCs. At least as current code, I think you can remove it.